mdb-realm/src/deploy.ts

```diff
@@ -0,0 +1,28 @@
+import * as core from "@actions/core";
+import { createCluster, getConfig } from "./helpers";
+
+async function run(): Promise<void> {
+    try {
+        const config = getConfig();
+        for (let i = 0; i < 5; i++) {
+            try {
+                await createCluster(config);
+                core.setOutput("clusterName", config.clusterName);
+                core.setOutput("atlasUrl", config.atlasUrl);
+                core.setOutput("realmUrl", config.realmUrl);
+
+                return;
+            } catch (e) {
+                core.warning(`An error occurred while deploying cluster '${config.clusterName}': ${e}. Retrying...`);
+            }
+        }
+
+        core.setFailed(`Failed to deploy ${config.clusterName} after 5 attempts.`);
```

Comment:
Better pull this into its own variable an use it for the loop too.
```suggestion
        core.setFailed(`Failed to deploy ${config.clusterName} after ${ATTEMPTS} attempts.`);
```